### PR TITLE
add noah's author file

### PR DESCRIPTION
**Pull request by n-dohrmann**
### Description
Add Noah Dohrmann author file for OSCI 2023
 
### Check List

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/noahdohrmann.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Noah Dohrmann
+short_name:
```
Comment (cwperks) [inline _authors/osci_2023/noahdohrmann.markdown:3]: Please remove this line if its not applicable

File: _authors/osci_2023/noahdohrmann.markdown [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+linkedin: www.linkedin.com/in/noah-dohrmann
```
Comment (cwperks) [inline _authors/osci_2023/noahdohrmann.markdown:4]: Please only include the username and not the full linkedin URL

File: _authors/osci_2023/noahdohrmann.markdown [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+github: n-dohrmann
+---
+
+**<Noah Dohrmann>** <Hi! Happy to get started here!>
```
Comment (cwperks) [inline _authors/osci_2023/noahdohrmann.markdown:8]: Please remove the angled brackets <> from the placeholder

**Review by cwperks — CHANGES_REQUESTED** [review]

File: _authors/osci_2023/noahdohrmann.markdown [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+name: Noah Dohrmann
+linkedin: noah-dohrmann
+github: n-dohrmann
+---
+
+**<Noah Dohrmann>** Hi! Happy to get started here!
```
Comment (cwperks) [inline _authors/osci_2023/noahdohrmann.markdown:7]: One last request, can you remove the angled brackets <> around your name on this line?

Comment (n-dohrmann) [reply _authors/osci_2023/noahdohrmann.markdown:7]: Yep! Sorry, I was confused about the syntax there. Have a good evening!

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**